src/components/ToastCard/_toast-card.scss

```diff
@@ -0,0 +1,35 @@
+@import "vanilla-framework/scss/vanilla";
+
+.toast-card {
+  background-color: $color-x-light;
+  border: 1px solid $color-mid-light;
+  border-radius: 5px;
+  box-shadow: 0 0 2rem 0 rgba(51, 51, 51, 0.2);
```

Comment:
Do we have a variable for a drop shadows throughout the app?

Reply:
Seems `$box-shadow` is available in Vanilla 🤙🏻